subprojects/gradle/slack/build.gradle.kts

```diff
@@ -30,5 +30,7 @@ tasks.named<Test>("integrationTest").configure {
 }
 
 fun Test.applyOptionalSystemProperty(name: String) {
-    project.property(name)?.toString()?.let { value -> systemProperty(name, value) }
+    if(project.hasProperty(name)) {
+        project.property(name)?.toString()?.let { value -> systemProperty(name, value) }
+    }
 }
```

Comment:
Can we reuse the function between modules?

Reply:
I don't know how yet

Reply:
just extracted this function to convention plugin directory and it seems working